app/site/app/assets/stylesheets/site/application.scss

```diff
@@ -2,3 +2,7 @@
 @import 'modules/typo';
 @import 'modules/layout';
 @import 'components/data-header';
+
+.menu-item a, .menu-heading a {
```

Comment:
This shouldn't really be in the main application scss file IMO. What does it belong to?